### PR TITLE
Update maintainers.md

### PR DESCRIPTION
**Pull request by odow**
A few I forgot to update here, as well as some new packages. Going forward, I'll remember to update this list when adding the maintainer.

### CODE REVIEW

**Review by mlubin — APPROVED** [review]

File: pages/maintainers.md
Comment (odow) [inline pages/maintainers.md:40]: I'll wait for @jd-foster to accept the invite before merging this.

Comment (jd-foster) [reply pages/maintainers.md:40]: Thank you Oscar for the invitation to be a maintainer! I have accepted.

**Pull request merged**
